src/Mvc/Mvc.Analyzers/src/Startup/StartupAnalyzer.Events.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
Super unrelated, we should consider renaming the project. The name's pretty immaterial to users now that they never see it, but we'd likely use it for entirely non-MVC scenarios now

Reply:
I'm in favour of us shipping one analyzer for ASP.NET Core and one for Microsoft.Extensions. 

I don't want to make this part of james work.